core/src/smartcontracts/isi/permissions/has_token.rs

```diff
@@ -62,7 +62,8 @@ impl<H: HasToken> IsAllowed for HasTokenAsValidator<H> {
         };
 
         let contain = match wsv.map_account(authority, |account| {
-            account.contains_permission(&permission_token)
```

Comment:
should we consider removing `contains_permissions` from API?

Reply:
Yeah, I was thinking about that. But firstly, I believe, we should rethink how do we store permission tokens. Take a look here: #2482